### PR TITLE
Completed javascript-timers

### PR DESCRIPTION
**Pull request by amorabryan**
Interval
Timeout

### CODE REVIEW

**Comment by amorabryan** [comment]
Timeout
Interval

File: javascript-timers/interval.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+let count = 4;
+
+const countdown = function startCountdown() {
+  const $countdownDisplay = document.querySelector('.countdown-display');
```
Comment (shawnkost) [inline javascript-timers/interval.js]: This should be done globally. By including it inside the function, you're forcing us to traverse the DOM over and over every single time this function gets called

File: javascript-timers/interval.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  count--;
+  $countdownDisplay.textContent = count;
+  if (count < 1) {
+    $countdownDisplay.textContent = '~Earth Beeeelooowww Us~';
+    clearInterval(countdown);
```
Comment (shawnkost) [inline javascript-timers/interval.js]: You're passing in the wrong argument here. The argument should be an id that gets returned by the `setInterval` function

File: javascript-timers/timeout.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+function delay() {
+  const $message = document.querySelector('.message');
```
Comment (shawnkost) [inline javascript-timers/timeout.js]: Move this globally rather than inside the function

**Review by shawnkost — CHANGES_REQUESTED** [review]
A few changes here Bryan!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**